compiler/test/suites/includes.re

```diff
@@ -207,4 +207,38 @@ describe("includes", ({test, testSkip}) => {
     "from \"reprovideContents\" include ReprovideContents; use ReprovideContents.{ type OtherT as Other }; print({ x: 1 }: Other)",
     "{\n  x: 1\n}\n",
   );
+  /* Duplicate imports */
+  test("dedupe_includes", ({expect}) => {
+    let name = "dedupe_includes";
+    let outfile = wasmfile(name);
+    ignore @@
+    compile(
+      ~hook=Grain.Compile.stop_after_assembled,
+      name,
+      {|
+      module DeDupeIncludes
+      // Ensures fd_write is only included once
+      print("test")
```

Comment:
Can you make this test more obvious by literally importing an external function twice?